pkg/kube/containers.go

```diff
@@ -35,6 +35,37 @@ func ContainerForController(ds types.DeploymentSpec) corev1.Container {
 		ImagePullPolicy: GetPullPolicy(ds.Image.PullPolicy),
 		Name:            types.ControllerContainerName,
 		Env:             ds.EnvVar,
+		Ports:           ds.Ports,
+		StartupProbe: &corev1.Probe{
```

Comment:
Maybe, in an separate PR/Issue, this could also be done as part of a `skupper update` to existing deployments.

Reply:
Good point, updated now